custom_components/zaptec/zaptec/zconst.py

```diff
@@ -194,3 +194,7 @@ def type_user_roles(self, val: int) -> str:
         # v != 0 is needed to avoid the 0 == 0 case leading to None always being included
         roles = {k for k, v in self.get("UserRoles", {}).items() if v != 0 and (v & val) == v}
         return ", ".join(roles)
+
+
+ZCONST = ZConst()
+"""Zaptec constants instance."""
```

Comment:
While this might be more accurate than the old "Global var for the API constants from Zaptec"-explanation, the old one at least gave me a clearer impression of what it was for and where it came from.

I think it would be worth it to expand the docstring a bit here, considering this is also used outside of the API-folder and it's nice to get that explanation on a mousover:
<img width="596" height="251" alt="image" src="https://github.com/user-attachments/assets/74eca477-ddf3-41d7-a353-86674c3df931" />